src/sdltiles.cpp

```diff
@@ -61,6 +61,10 @@ static bool needupdate = false;
 extern bool tile_iso;
 extern WINDOW *w_hit_animation;
 
+// used to replace SDL_RenderFillRect with a more efficient SDL_RenderCopy
+SDL_Texture* alternate_rect_tex = NULL;
```

Comment:
Please avoid raw pointers, use `SDL_Texture_Ptr` instead - it's a `unique_ptr`. You can also unconditionally call `reset()` on it to delete the contained pointer (no need to check for `nullptr` before doing so).

Reply:
Thanks. Raw pointers were in the original PR and I didn't change them to unique pointers. I will change that now.